cmake/dependencies.cmake

```diff
@@ -304,13 +304,15 @@ if(ENABLE_INTEL_GNA)
             LIST(APPEND FILES_TO_EXTRACT_LIST gna_${GNA_VERSION}/linux)
         endif()
 
+        set(IE_PATH_TO_DEPS "http://10-211-120-125.iotg.sclab.intel.com/dl_score_engine")
```

Comment:
BTW, current GNA library does not work on Ubuntu 18. It requires glic 2.29, while u18 has glic 2.27
Could you please provide proper libraries which works on U18 and higher?